examples/basics/ontologies.ipynb

```diff
@@ -10,9 +10,61 @@
     "* An ontology defines the types of vector tools and classifications available for a project\n",
     "* Currently the only way to create an ontology object is by setting up a project with one.\n",
     "* This notebook will cover client side tools for working with ontologies\n",
-    "* Project setup is explained here: https://github.com/Labelbox/labelbox-python/blob/develop/examples/project_configuration/project_setup.ipynb\n",
-    "-----\n",
-    "Set the following variable to run this notebook"
+    "* Project setup is explained here: https://github.com/Labelbox/labelbox-python/blob/develop/examples/project_configuration/project_setup.ipynb\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "continuous-syndrome",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install labelbox"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "passing-question",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from labelbox.schema.ontology import OntologyBuilder, Tool, Classification, Option\n",
```

Comment:
can you import everything from just `label box`? why import from `labels.schema.ontology`

Reply:
You can't and I could change this to be top level but there could be a name conflict at some point. Classification here is a classification feature schema.